### DISCUSSION TITLE
Can't get bento-accordion compress/expand state persistence to work

### DISCUSSION THREAD

**Discussion by BrucePerens** [ampproject/bento#11] [Q&A]
Hi,

The bento-accordion documentation says: _A Bento Accordion with a defined id preserves the collapsed or expanded state of each section while the user remains on your domain._

Can anyone show me a demo in which this works. It's not working on the page at https://bentojs.dev/components/bento-accordion/
with my recent Chrome browser.

Thanks

Bruce

**Comment by danielrozenberg** [comment]
@ampproject/wg-components
